test/examples/src/readme_suite_test.go

```diff
@@ -0,0 +1,100 @@
+/**
+ * © Copyright IBM Corporation 2022, 2023. All Rights Reserved.
```

Comment:
```suggestion
 * © Copyright IBM Corporation 2025. All Rights Reserved.
```

Reply:
The consequences of not willing to write ginkgo suite scaffolding from a scratch. Fixed in https://github.com/IBM/cloudant-go-sdk/pull/701/commits/f152b674141b227b775c4176cf037712bc725edb